luigi/contrib/sqla.py

```diff
@@ -160,11 +160,10 @@ class SQLAlchemyTarget(luigi.Target):
     to create a task to write to the database.
     """
     marker_table = None
-    connect_args = {}
     _engine_dict = {}  # dict of sqlalchemy engine instances
     Connection = collections.namedtuple("Connection", "engine pid")
 
-    def __init__(self, connection_string, target_table, update_id, echo=False, connect_args=None):
+    def __init__(self, connection_string, target_table, update_id, echo=False, connect_args={}):
```

Comment:
One suggestion I have here is to leave `connect_args` default value as `None` and assign inside the function as
`self.connect_args = {} if connect_args is None else connect_args`

This is to avoid having mutables as default values in function parameters.